src/test/component.test.js

```diff
@@ -0,0 +1,20 @@
+import { test, expect } from '@jest/globals';
+
+test('Check if a home cards are  displayed home page properly', () => {
+  const number = 24;
+
+  const isEven = number % 2 === 0;
+
+  if (isEven) {
+    expect(isEven).toBe(true);
+  } else {
+    expect(isEven).toBe(false);
+  }
+});
```

Comment:
- [ ] Just as mentioned earlier, Please test your application functionalities, This tests are not related to the project.

Reply:
- [ ] Dear @enocol, I can assure you I have checked all the files. I could not find enough tests to assess the functionality of your application or the redux actions. Please mention the tests you think will explain the functionality of the application and the next reviewer will approve. Also, it's still possible for you to ask for a second view of the project if you think I'm being unfair here. You got this 👍

Reply:
I have made new Test and I am exhausted now.

Reply:
- [ ] Dear @enocol, being exhausted is our day-to-day life as a programmer, Please calm down and try to address the issue. We just have to address all the requirements as it's expected in the real world. I've been there, and I know you can overcome this.